docs/source/_templates/layout.html

```diff
@@ -9,23 +9,23 @@
   }}();
 </script> -->
 
-<title>EvalML - Alteryx Open Source AutoML Library</title>
-<meta name="description" content="EvalML is an AutoML library that builds, optimizes, and evaluates machine learning pipelines using domain-specific objective functions. — EvalML documentation">
+<title>CheckMate - Alteryx Open Source AutoML Library</title>
+<meta name="description" content="CheckMate is an AutoML library which catches and warns of problems with your data and problem setup before modeling. - CheckMate documentation">
```

Comment:
Not an `AutoML` library, maybe change to something like above

Reply:
refactored to fit open source library